themes/default/content/blog/pulumi-release-notes-0.57/index.md

```diff
@@ -0,0 +1,114 @@
+
+<!--more-->
+
+## New and updated cloud providers
+
+### Pulumi supports Amazon ECS Anywhere and Amazon App Runner
+
+Earlier this month, Amazon announced a [new version of their Elastic Container Service](https://aws.amazon.com/blogs/containers/introducing-amazon-ecs-anywhere/) that allows you to deploy containers to almost any server and a new fully-managed container-native service called [AWS App Runner](https://aws.amazon.com/blogs/containers/introducing-aws-app-runner/). Pulumi had full same-day support for both new services.
+
+Learn how to get started with [ECS Anywhere]({{< relref "/blog/ecs-anywhere-launch" >}}) and [App Runner]({{< relref "/blog/deploy-applications-with-aws-app-runner" >}}).
+
+### Pulumi supports Sumo Logic
+
+This milestone, we released a new provider for Sumo Logic. You can now manage Sumo Logic's big data and analytics services with Pulumi infrastructure-as-code.
+
+Get started in the [SumoLogic provider docs]({{< relref "/docs/intro/cloud-providers/sumologic" >}}).
+
+### 100 new resources in the Azure Native provider
+
+In this milestone, we shipped 3 new versions of the Azure Native provider (1.10.0, 1.9.0, and 1.8.0) that collectively added 100 more Azure resources that you can manage with the native provider. These include new analytics, machine learning, and security capabilities across the breadth of the Azure platform.
+
+[See the full list](https://github.com/pulumi/pulumi-azure-native/blob/v1.10.0/CHANGELOG.md)
+
+## Pulumi engine and core technologies
+
+In this milestone, we shipped Pulumi versions [3.3.0](https://github.com/pulumi/pulumi/releases/tag/v3.3.0), [3.3.1](https://github.com/pulumi/pulumi/releases/tag/v3.3.1), and [3.4.0](https://github.com/pulumi/pulumi/releases/tag/v3.4.0). This section highlights key changes in those versions; for the full list, click the links for each version to visit the changelog for that version.
+
+### Pulumi Packages schema documentation now available
+
+Pulumi Packages, released at [PulumiUP earlier this year]({{< relref "/blog/pulumiup-pulumi-packages-multi-language-components" >}}) enable you to share Pulumi components with your colleagues and with the Pulumi community. But previously, it could be difficult to [author a Pulumi Package]({{< relref "/docs/guides/pulumi-packages" >}}) because the schema you use to define your package wasn't fully documented. Now, comprehensive schema documentation is available [in the docs]({{< relref "/docs/guides/pulumi-packages" >}}).
+
+### Pulumi Packages schemas can reference other packages' types
+
+For more complex Pulumi Package scenarios, you may want to be able to reference the types in one Pulumi Package from another Pulumi Package. Previously, this was only possible if you authored your Pulumi Package in Go; now, it's possible in all Pulumi Languages.
+
+[#6777](https://github.com/pulumi/pulumi/issues/6777)
+
+### Input parameters can now be marked as secret
```

Comment:
Suggest we can combine the three sections above into a single section on improvements to Pulumi Packages.